draft-ietf-mls-protocol.md

```diff
@@ -1273,6 +1273,7 @@ struct {
             opaque application_data<0..2^32-1>;
     }
 
+    opaque authenticated_data<0..2^32-1>;
```

Comment:
Nit: I would put this just below the content_type field, just to preserve parallelism (`MLSCiphertext.ciphertext -> MLSPlaintext.operation/application_data`)

Reply:
Also it would avoid interleaving plaintexts and ciphertexts...

Reply:
I agree. Done!